src/main/diagnostics/abstractCodeActionProvider.ts

```diff
@@ -111,4 +108,18 @@ export abstract class AbstractCodeActionProvider implements vscode.CodeActionPro
         this._diagnosticCollection.clear();
         this._diagnosticCollection.dispose();
     }
+
+    protected getSeverityMessage(node: DependenciesTreeNode): string {
```

Comment:
```suggestion
    private getSeverityMessage(node: DependenciesTreeNode): string {
```

Reply:
Following our slack conversation. I have removed this method